src/test/java/org/jabref/gui/journals/ManageJournalAbbreviationsViewModelTest.java

```diff
@@ -57,15 +57,15 @@ void setUpViewModel(@TempDir Path tempFolder) throws Exception {
         TaskExecutor taskExecutor = new CurrentThreadTaskExecutor();
         JournalAbbreviationLoader journalAbbreviationLoader = mock(JournalAbbreviationLoader.class);
         viewModel = new ManageJournalAbbreviationsViewModel(preferences, dialogService, taskExecutor, journalAbbreviationLoader);
-        emptyTestFile = createTestFile(tempFolder, "emptyTestFile.txt", "");
-        testFile1Entries = createTestFile(tempFolder, "testFile1Entries.txt", "Test Entry = TE" + NEWLINE + "");
-        testFile3Entries = createTestFile(tempFolder, "testFile3Entries.txt",
-                "Abbreviations = Abb" + NEWLINE + "Test Entry = TE" + NEWLINE + "MoreEntries = ME" + NEWLINE + "");
-        testFile4Entries = createTestFile(tempFolder, "testFile4Entries.txt", "Abbreviations = Abb" + NEWLINE
-                + "Test Entry = TE" + NEWLINE + "MoreEntries = ME" + NEWLINE + "Entry = E" + NEWLINE + "");
-        testFile5EntriesWithDuplicate = createTestFile(tempFolder, "testFile5Entries.txt",
-                "Abbreviations = Abb" + NEWLINE + "Test Entry = TE" + NEWLINE + "Test Entry = TE" + NEWLINE
-                        + "MoreEntries = ME" + NEWLINE + "EntryEntry = EE" + NEWLINE + "");
+        emptyTestFile = createTestFile(tempFolder, "emptyTestFile.csv", "");
```

Comment:
Please dublicate the test:

One for a CSV file without shortest unique, one for a CSV file with shortest unique, one CSV file with mixed entries (some with shortest unique, some without).

Reply:
I think, this is still a TODO.

Reply:
Sure, that is the reason because I did not resolve this conversation. I am working on it, step by step... :smiley: